doc/en/user/source/configuration/tools/bulk/index.rst

```diff
@@ -26,7 +26,7 @@ Duplicating Configuration
 
 5. Choose to recursively copy:
 
-   * Recursively copying a worksapce will duplicate all stores and layers contained in the workspace
+   * Recursively copying a workspace will duplicate all stores and layers contained in the workspace
    * Recursively copying a store will copy all layers published by the store
    
-6. Press :guilabel:`Start` to being duplicating
\ No newline at end of file
+6. Press :guilabel:`Start` to being duplicating
```

Comment:
I guess this would have been "begin" instead of "being"? Just noticed it because github reported a change on this line, and I could not find it :-D (probably a newline difference)

Reply:
Yes, thank you Andrea.  Fixed